nix/docker.nix

```diff
@@ -13,7 +14,7 @@ let
   args = {
     name = image-name;
     created = "now";
-    contents = [ package ];
+    contents = [ package pkgs.cacert ];
```

Comment:
not sure how this works, but the rest looks good to me. will approve under the assumption that this will all be handled automagically.

Reply:
Yeah the docker container needs cacert for TLS. Tested locally.